.github/workflows/actions.yaml

```diff
@@ -0,0 +1,24 @@
+name: test
+on: [push]
+jobs:
+#  lint_check: 
+#    runs-on: ubuntu-latest
+#    steps:
+#      - name: copy stuff into the repo
+#        uses: actions/checkout@v1
+#      - name: install linter
+#        run: sudo apt-get install -y pycodestyle
+#      - name: vibe check
+#        run: pycodestyle app.py
+  main_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: copy stuff from repo to worker directory
+        uses: actions/checkout@v1
+      - run: sudo apt-get update
+      - run: sudo apt-get install -y python-pytest python-requests
```

Comment:
The GitHub Actions runner comes with python installed - rather than using `apt`, you can (and probably should) use `pip` to install pytest and requests

Reply:
oh neat, will do